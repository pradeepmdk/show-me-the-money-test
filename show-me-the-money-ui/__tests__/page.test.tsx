import Page from '@/app/page'
import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer';

describe('Page', () => {
  it('renders a component', () => {
    const component = renderer.create(
      <Page />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})