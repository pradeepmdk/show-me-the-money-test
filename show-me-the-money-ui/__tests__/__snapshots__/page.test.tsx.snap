// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders a component 1`] = `
<main
  className="flex min-h-screen flex-col  justify-between p-24"
/>
`;

exports[`Page renders a heading 1`] = `
<main
  className="flex min-h-screen flex-col  justify-between p-24"
/>
`;
